### DISCUSSION TITLE
Support GCP?

### DISCUSSION THREAD

**Discussion by prawee** [refinedev/refine#1589] [Q&A] [answered]
Do you plan to make example for deploy to google cloud?
such as
- Google App Engine
- Google Cloud Run

Thank you.

**Comment by omeraplak** [comment, marked as answer]
Hey @prawee ,
Refine apps are typical React and Next.js apps. Do we really need an example for GCP?

**Reply by prawee** [reply]
Yes i want it. I try it by myself. It's not working.

Thank you for your reply.
